src/generate_reduce.jl

```diff
@@ -0,0 +1,21 @@
+"""
+    generate_min_set(g, gen_func, Reps)
+Generate a vector `Reps` times using `gen_func(g)` and return the vector with the least elements.
+"""
+generate_min_set(g::AbstractGraph{T}, gen_func, Reps::Integer) where T<: Integer =
+mapreduce(gen_func, (x, y)->length(x)<length(y) ? x : y, Iterators.repeated(g, Reps))
+
+"""
+    generate_max_set(g, gen_func, Reps)
+Generate a vector `Reps` times using `gen_func(g)` and return the vector with the most elements.
+"""
+generate_max_set(g::AbstractGraph{T}, gen_func, Reps::Integer) where T<: Integer =
+mapreduce(gen_func, (x, y)->length(x)>length(y) ? x : y, Iterators.repeated(g, Reps))
```

Comment:
Won't this be equal to `gen_func(g)`?

Reply:
Yes.
The idea is to run `gen_func(g)` `Reps` times in parallel.

It's a random algorithm so it will return different values for each call.

So out of the `Reps` instances of `gen_func(g)`, return the largest instance.

Reply:
Ok!